packages/components/src/components/collapsible/collapsible.tsx

```diff
@@ -48,9 +51,10 @@ const Collapsible = ({
     React.useEffect(
         () =>
             setShouldShowCollapsible(
-                React.Children.toArray(children).some(({ props }: any) => 'data-collapsible' in props)
+                React.Children.toArray(children).some(({ props }: any) => 'data-collapsible' in props) ||
+                    hide_collapsible_button
```

Comment:
especially here. show_collapsible_button would make it easier to think about the logic